### PR TITLE
Purchase ticket UI implement in progress

### PR DESCRIPTION
**Pull request by Tobipeters**
- Reuseable modal component 
- -Purchase ticket component UIs

### CODE REVIEW

File: src/assets/svg.tsx
Comment (TemitopeAsama) [inline src/assets/svg.tsx:1]: I don't understand why you created a custom SVG component when SVGs can simply be imported as React components?

File: package.json
Comment (TemitopeAsama) [inline package.json:1]: Please use custom ports on your end when running the dev command. Let's keep it at 3000 on the project.

File: src/components/modal/modal.module.scss
Comment (TemitopeAsama) [inline src/components/modal/modal.module.scss:1]: Please use the breakpoints from the _brakpoints.scss files to reduce ambiguity. Instantiate your custom breakpoints on the root stylesheet.

File: src/components/purchase-ticket/components/checkout/checkout.module.scss
Comment (TemitopeAsama) [inline src/components/purchase-ticket/components/checkout/checkout.module.scss:1]: Fix issue with breakpoints here too.

File: src/components/purchase-ticket/ticket.module.scss
Comment (TemitopeAsama) [inline src/components/purchase-ticket/ticket.module.scss:1]: Issue with breakpoints here too.

Comment (TemitopeAsama) [inline src/components/purchase-ticket/ticket.module.scss:1]: Also, I don't see the forms and all.

**Review by TemitopeAsama — CHANGES_REQUESTED** [review]
Please fix issues as outlined as soon as you can. Thanks.

File: src/assets/svg.tsx
Comment (Tobipeters) [reply src/assets/svg.tsx:1]: This has been resolved

File: package.json
Comment (Tobipeters) [reply package.json:1]: Resolved

File: src/components/modal/modal.module.scss
Comment (Tobipeters) [reply src/components/modal/modal.module.scss:1]: Resolved

File: src/components/purchase-ticket/components/checkout/checkout.module.scss
Comment (Tobipeters) [reply src/components/purchase-ticket/components/checkout/checkout.module.scss:1]: Resolved

File: src/components/purchase-ticket/ticket.module.scss
Comment (Tobipeters) [reply src/components/purchase-ticket/ticket.module.scss:1]: Resolved

Comment (Tobipeters) [reply src/components/purchase-ticket/ticket.module.scss:1]: Work in progress

**Review by TemitopeAsama — APPROVED** [review]

**Pull request merged**